src/Mono.Android/Mono.Android.targets

```diff
@@ -128,6 +129,7 @@
         SourceFiles="$(IntermediateOutputPath)__dex\classes.dex"
         DestinationFiles="$(OutputPath)mono.android.dex"
     />
+    <Touch Files="$(OutputPath)mono.android.dex" />
```

Comment:
Well, I was running it from my o-preview tree where *I* removed them by myself. Your grep results also say that they are not REFERENCED.